locales/en.json

```diff
@@ -7,6 +7,20 @@
   "wmo": "WMO",
   "wmoFull": "World Meteorological Organization",
   "woudcFull": "World Ozone and Ultraviolet Radiation Data Centre",
+  "map-instructions": {
+    "label": "How to Use: Interactive Map",
+    "panning": "Panning:",
+    "tab": "Tab",
+    "template": "{panning} To pan the map, click and drag over the interactive map. Alternatively, {tab} focus to the interactive map box and then press the arrow keys.\n\n{zooming} To zoom in and out of the map, mouse scroll up and down while you mouse over the interactive map or click the {plus} and {minus} buttons respectively. Alternatively, {tab} focus to the interactive map box and then press the {plus} or {minus} keys. To zoom out to the maximum map extent, press the \"zoom to globe\" button.",
+    "zooming": "Zooming:"
+  },
+  "table-instructions": {
+    "filtering": "Filtering:",
+    "label": "How to Use: Interactive Table",
+    "paging": "Paging:",
+    "sorting": "Sorting:",
+    "template": "{filtering} To filter the table, type in the search bars at the top of each column or the \"search all\" bar.\n\n{sorting} To sort the table, click on the arrow symbols next to the column headings.\n\n{paging} Change the \"show entries\" dropdown to change how many stations are displayed on the page. Click \"next\" or \"previous\" to show additional pages of the table."

```

Comment:
Good idea on naming the templates after splitting them up. I worry that it will be inapplicable some of the time and some keys would be left as `"template"` even if other are changed, is it better to have some more informative names or to keep the names consistent as `"template"` across the board?

Getting rid of newlines and splitting up the paragraphs into multiple... uh, paragraphs was going to part of the next pull request and all this falls in that area. Is that okay?

Reply:
Makes sense. We can do the paragraphs fix in another PR.

Existing names of "template": I'll leave that to you to decide and adjust accordingly on a case by case basis. I find that when I create deeper nested objects, it can be tricky to name the keys sometimes while still making meaningful sense at a glance. For better simplicity, my route is generally to use shallower nesting but more meaningful key names.